js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,16 @@
+function aperture(tupleLength, array) {
+  const result = [];
+  for (let i = 0; i < array.length; i++) {
```

Comment:
There should be a condition before FOR loop to check if the array length is less than tuple length, if yes then return []. We should not let FOR & WHILE loop execute in this case.